code/game/objects/effects/contraband.dm

```diff
@@ -657,6 +657,11 @@
 	desc = "This outdated informational poster uses Safety Moth(TM) to tell the viewer to hide in lockers when the Supermatter Crystal has delaminated. Running away might be a better strategy. It's signed by 'AspEv'."
 	icon_state = "poster44_legit"
 
+/obj/structure/sign/poster/official/disk
+	name = "The Nuke Disk"
+	desc = "This poster is designed for the captain, as a friendly reminder to secure the disk. It's signed by 'ether'."
```

Comment:
Do not sign the poster, please...

Reply:
narcissistic lol

Reply:
nah, the other ones are signed so it's fair.